app/models/WhichBestDescribesYou.scala

```diff
@@ -48,7 +48,7 @@ object WhichBestDescribesYou {
     }
   }
 
-  def isBusinessRepresentative(answers: UserAnswers): Boolean = {
+  def theUserIsAnAgent(answers: UserAnswers): Boolean = {
```

Comment:
`isSignedInUserAnAgent`

Reply:
`if(is...)` never reads very well in my opinion